src/core/Global.ttslua

```diff
@@ -645,6 +646,16 @@ function emptyChaosBag()
   end
 end
 
+-- find all chaos tokens that are on the table to the chaos bag (e.g. sealed on cards)
+function findAllChaosTokens()
+  local chaosbag = findChaosBag()
+  for _, obj in ipairs(getObjects()) do
+    if tokenChecker.isChaosToken(obj) and not obj.hasTag("tempToken") then
+      chaosbag.putObject(obj)
```

Comment:
If this finds Bless/Curse and puts them in the bag, will it leave the B/C manager in an incorrect state?

Reply:
The bless/curse manager indexes these tokens `onLoad()`. But if this is called it would probably leave it in a wrong state 🤔

Reply:
Updated it to exclude bless / curse tokens (since these are temporary)